README.md

```diff
@@ -71,16 +71,19 @@ func _process(delta):
     Rakugo.do_step()
 ```
 
+- Create a text file "Timeline.rk" in your project folder (res://)
```

Comment:
@theludovyc This sounds like they can only create one file `rk` file *res://Timeline.rk*

Reply:
Think something like this will just do (I use `*` to make markdown italic):
> Create your first RakuScript (text file) *"Timeline.rk"*  where ever you want inside your project folder (*res://*)

Reply:
Got it :) !